2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,14 +54,32 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>
+                     <persName ref="PRS3082Chaine"/> was mistakenly identified this a malkǝʾ-hymn for
```

Comment:
delete `was`

Reply:
rephrase the English (`Chaîne mistakenly identified this malk... as the one for ... `or `Chaîne mistakenly identified the addressee of the hymn as ...` or anything else that would make sense)